test/unit/utils_test.js

```diff
@@ -1,26 +1,26 @@
 const randomBytes = require("randombytes");
 
 function newClientSigner(key, weight) {
-  return { key, weight }
+  return { key, weight };
```

Comment:
It doesn't seem like anything changes in these tests as a result of this change, which shouldn't be the case when we add a possible error case to a function (home domain doesn't match).

Can we have tests that pass invalid `homeDomain` parameters and expect a failure? And then have success cases? We have at least a handful of tests in the Go SDK that test the functionality that changes here: 

https://github.com/stellar/go/pull/3182/files#diff-6b43f5d421fd11237c44ed17f19e8f18732558111da0f29f6cbbf0ea09d23fb2R2040

https://github.com/stellar/go/pull/3182/files#diff-6b43f5d421fd11237c44ed17f19e8f18732558111da0f29f6cbbf0ea09d23fb2R2512

Reply:
We now have home domain success and failure tests, do we need anything else?